Automattic/woocommerce-services
# Remove/stripe connect functionality

## Description

This removes Stripe onboarding functionality from the plugin altogether.

This is part of the work to move Stripe OAuth code from WC Services over to Stipe. For more information see pb0GrA-Pd-p2

### Testing Instructions

The changes in this PR are dependent on https://github.com/woocommerce/woocommerce-admin/pull/4933. 

Once the above is merged, you can test this by installing WCS and ensuring non-stripe functionality is not broken. For good measure, you can also check that everything works as expected with the various repos from the above P2 checked out locally.

I was unsure about how to test the removal of Stripe code from `client/extensions/woocommerce` directory, so I've removed this in a separate commit just in case this has an impact on something like WordPress.com: https://github.com/Automattic/woocommerce-services/commit/77a209c68cdc0d0ee346117a49c7cab73b379b28

I also noticed Stripe connect code exists in the `wp-calypso` submodule. I haven't opened a PR there to remove this as I wanted to get feedback here first. I'd love some input on this if at all possible.

### Checklist


One unit test is failing here and on `develop`:

```
 ShippingRate › for all rates › renders the delivery date

    Expected value   <div className="rates-step__shipping-rate-delivery-date">January 1</div>
    Received:
      {}
```

As this test is unrelated to these changes and exists on `develop` I have pushed with `--no-verify` flag.
